src/components/SourcesTree.js

```diff
@@ -54,6 +55,15 @@ let SourcesTree = React.createClass({
       return this.setState({ listItems });
     }
 
+    if (nextProps.hlightSourceURL != this.props.hlightSourceURL) {
+      const hLightItems = getDirectories(
+        nextProps.hlightSourceURL,
+        this.state.sourceTree
+      );
+
+      return this.setState({ hLightItems });
+    }
```

Comment:
is it possible to just highlight the source? 

I'm asking because we should know that it's already expanded